firmware/src/can.cpp

```diff
@@ -24,73 +24,66 @@ extern uint8_t sfocCmdStr;
 static void handleCanMessage(FDCAN_RxHeaderTypeDef rxHeader, uint8_t *rxData);
 static void configureCAN();
 
-SimpleCan can1(-1,-1);
-SimpleCan::RxHandler can1RxHandler(8, handleCanMessage);
-
 FDCAN_TxHeaderTypeDef TxHeader;
 uint8_t TxData[8];
 
-static void configureCAN(){
```

Comment:
Splitting the ternary like this feels weird, but also I don't super care if we change it.